<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>SlideShowCfg</class>
  <widget class="QWidget" name="SlideShowCfg" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>243</height>
      </rect>
    </property>
    <layout class="QGridLayout" >
      <property name="margin" >
        <number>0</number>
      </property>
      <item rowspan="1" row="7" column="0" colspan="4" >
        <widget class="QCheckBox" name="mCbxSubdirectory" >
          <property name="text" >
            <string>&amp;Include images from sub-folders</string>
          </property>
        </widget>
      </item>
      <item rowspan="1" row="1" column="0" colspan="3" >
        <widget class="QCheckBox" name="mCbxRandom" >
          <property name="text" >
            <string>&amp;Random order</string>
          </property>
        </widget>
      </item>
      <item rowspan="1" row="6" column="2" colspan="3" >
        <widget class="KUrlRequester" name="mDirChooser" />
      </item>
      <item rowspan="1" row="0" column="0" colspan="3" >
        <widget class="QCheckBox" name="mCbxZoom" >
          <property name="text" >
            <string>Resi&amp;ze images</string>
          </property>
        </widget>
      </item>
      <item row="7" column="4" >
        <spacer name="spacer9" >
          <property name="sizeHint" >
            <size>
              <width>161</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Horizontal</enum>
          </property>
        </spacer>
      </item>
      <item rowspan="1" row="6" column="0" colspan="2" >
        <widget class="QLabel" name="textLabel1" >
          <property name="text" >
            <string>I&amp;mage folder:</string>
          </property>
          <property name="buddy" stdset="0" >
            <cstring>mDirChooser</cstring>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item rowspan="1" row="2" column="0" colspan="3" >
        <widget class="QCheckBox" name="mCbxShowName" >
          <property name="text" >
            <string>Show &amp;names</string>
          </property>
        </widget>
      </item>
      <item row="5" column="1" >
        <spacer name="spacer11" >
          <property name="sizeHint" >
            <size>
              <width>20</width>
              <height>19</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Vertical</enum>
          </property>
        </spacer>
      </item>
      <item rowspan="1" row="4" column="1" colspan="2" >
        <widget class="QSpinBox" name="mDelay" >
          <property name="suffix" >
            <string> sec</string>
          </property>
          <property name="specialValueText" >
            <string/>
          </property>
          <property name="maximum" >
            <number>240</number>
          </property>
          <property name="minimum" >
            <number>1</number>
          </property>
        </widget>
      </item>
      <item row="4" column="0" >
        <widget class="QLabel" name="textLabel1_2" >
          <property name="text" >
            <string>&amp;Delay:</string>
          </property>
          <property name="buddy" stdset="0" >
            <cstring>mDelay</cstring>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item rowspan="1" row="3" column="0" colspan="3" >
        <widget class="QCheckBox" name="mCbxRandomPosition" >
          <property name="text" >
            <string>Random &amp;position</string>
          </property>
        </widget>
      </item>
      <item rowspan="6" row="0" column="3" colspan="2" >
        <widget class="QFrame" name="mPreview" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>7</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="frameShape" >
            <enum>NoFrame</enum>
          </property>
          <property name="frameShadow" >
            <enum>Plain</enum>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <includes>
    <include location="local" >klineedit.h</include>
    <include location="local" >kpushbutton.h</include>
  </includes>
  <connections>
    <connection>
      <sender>mCbxZoom</sender>
      <signal>toggled(bool)</signal>
      <receiver>mCbxRandomPosition</receiver>
      <slot>setDisabled(bool)</slot>
    </connection>
  </connections>
</ui>
